src/components/implementation/no_interface/scheddev/sl_mod_fprr.c

```diff
@@ -7,6 +7,8 @@
 #define SL_FPRR_HIGHEST 0
 #define SL_FPRR_LOWEST  (SL_FPRR_NPRIOS-1)
 
+#define SL_FPRR_US_MIN  1000
```

Comment:
I'm not sure what this is, but consider if it should be `#define SL_FPRR_US_MIN SL_PERIOD_US`...